test_data/asm/vm_to_block.asm

```diff
@@ -0,0 +1,75 @@
+machine Binary(latch) {
+
+    degree 8;
+
+    function and x, y -> z {
+    }
```

Comment:
What does the function body do?

Reply:
It is checked to be empty in the type checker. It's only there to keep some unity with VMs. With all these deviations maybe we should eventually bite the bullet and have a different syntax for block machines, but I feel this can be refactored later.